code/php/index/about.php

```diff
@@ -44,7 +44,8 @@
                     <p>We are a team of students in Software engineering and Computer engineering. Inspired by the idea of having a site dedicated to group projects, we decided to built a webstite that would contains all the features needed to work on the given project.</p>
             <button class="btn btn-default btn-lg">Get in Touch</button>
             <script>$("button").click(function () {
-                        $("#container").append('<div class="container-fluid" style="margin:0px 200px 0px 50px;"><h2 class="text-center">CONTACT</h2><div class="row"><div class="col-sm-5"><p>Contact us and we will get back to you within 24 hours.</p><p><span class="glyphicon glyphicon-map-marker"></span> Montreal, CANADA</p><p><span class="glyphicon glyphicon-phone"></span> 1800-123-4567 </p><p><span class="glyphicon glyphicon-envelope"></span> myemail@something.com</p> </div><div class="col-sm-7"><div class="row"><div class="col-sm-6 form-group"><input class="form-control" id="name" name="name" placeholder="Name" type="text" required></div><div class="col-sm-6 form-group"><input class="form-control" id="email" name="email" placeholder="Email" type="email" required></div></div><textarea class="form-control" id="comments" name="comments" placeholder="Comment" rows="5"></textarea><br><div class="row"><div class="col-sm-12 form-group"><button class="btn btn-default pull-right" type="submit">Send</button></div></div> </div></div></div>');});
+                        $("#container").append('<div class="container-fluid" style="margin:0px 200px 0px 50px;"><h2 class="text-center">CONTACT</h2><div class="row"><div class="col-sm-5"><p>Contact us and we will get back to you within 24 hours.</p><p><span class="glyphicon glyphicon-map-marker"></span> Montreal, CANADA</p><p><span class="glyphicon glyphicon-phone"></span> 1800-123-4567 </p><p><span class="glyphicon glyphicon-envelope"></span> myemail@something.com</p> </div><div class="col-sm-7"><div class="row"><div class="col-sm-6 form-group"><input class="form-control" id="name" name="name" placeholder="Name" type="text" required></div><div class="col-sm-6 form-group"><input class="form-control" id="email" name="email" placeholder="Email" type="email" required></div></div><textarea class="form-control" id="comments" name="comments" placeholder="Comment" rows="5"></textarea><br><div class="row"><div class="col-sm-12 form-group"><button class="btn btn-default pull-right" type="submit">Send</button></div></div> </div></div></div>');

```

Comment:
Any way we can make this line look a little nicer in terms of code style? It's pretty bad practice to have code which requires horiontal scrolling just to read.